src/app/nebular-testing.module.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  BUILT_IN_THEMES,
+  DEFAULT_MEDIA_BREAKPOINTS,
+  NB_BUILT_IN_JS_THEMES,
+  NB_DOCUMENT,
+  NB_JS_THEMES,
+  NB_MEDIA_BREAKPOINTS,
+  NB_THEME_OPTIONS,
+  NB_WINDOW,
+  NbJSThemesRegistry,
+  NbLayoutDirectionService,
+  NbLayoutRulerService,
+  NbLayoutScrollService,
+  NbMediaBreakpointsService,
+  NbOverlayContainerAdapter,
+  NbSpinnerService,
+  NbThemeService
+} from '@nebular/theme';
+
+import { NgModule } from '@angular/core';
+
+@NgModule({
+  providers: [
+    NbThemeService,
+    // Required by NbThemeService
+    NbMediaBreakpointsService,
+    { provide: NB_THEME_OPTIONS, useValue: { name: 'default' } },
+    { provide: NB_MEDIA_BREAKPOINTS, useValue: DEFAULT_MEDIA_BREAKPOINTS },
+    { provide: NB_JS_THEMES, useValue: [] },
+    { provide: NB_BUILT_IN_JS_THEMES, useValue: BUILT_IN_THEMES },
+    NbJSThemesRegistry,
+    // NbSpinnerService
+    NbSpinnerService,
+    // Required by NbLayoutComponent
+    { provide: NB_DOCUMENT, useValue: document },
+    { provide: NB_WINDOW, useValue: window },
```

Comment:
I really don't like using the real document and window here, but I'm unsure what else to do.  Creating mock objects to use instead would be better, but who knows what Nebular is reaching for on the Document or Window, so that's a rabbit hole for another day.